src/EditorFeatures/VisualBasicTest/DocumentationComments/DocumentationCommentTests.vb

```diff
@@ -1204,7 +1206,7 @@ End Class
         End Function
 
         Protected Overrides Function CreateTestWorkspace(code As String) As TestWorkspace
-            Return TestWorkspace.CreateVisualBasic(code)
+            Return TestWorkspace.CreateVisualBasic(code, exportProvider:=ExportProviderCache.GetOrCreateExportProviderFactory(TestExportProvider.EntireAssemblyCatalogWithCSharpAndVisualBasic.WithoutPartsOfType(GetType(CommitConnectionListener))).CreateExportProvider())
```

Comment:
@jasonmalinowski I thought you implemented the `CommitConnectionListener` somewhere else?

Reply:
/cc: @ivanbasov

Reply:
CommitConnectionListener is an existing file. VSSDK attempts to call within some unit test comparing to the previous version. To provide a unity of those tests, it would be better to exclude this class.

---
In reply to: [196966852](https://github.com/dotnet/roslyn/pull/27027#discussion_r196966852) [](ancestors = 196966852)